<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsForm</class>
 <widget class="QWidget" name="OptionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameBlue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>410</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36 , 41 , 67)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 26pt;
</string>
    </property>
    <property name="text">
     <string>Beállítások</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameButtons">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>520</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
